tools/train_net.py

```diff
@@ -67,7 +78,7 @@ def main():
     distributed = len(gpus) > 1
     device = torch.device('cuda:{}'.format(args.local_rank))
 
-    if distributed:
+    if distributed is False:
```

Comment:
It this a typo? Why initialize backend when `distributed is False`?

Reply:
I found that when ddp is not used, we need to copy the sample from the batch to the gpu manually, and it was just my mistake doing it in a opposite way.